### PR TITLE
upgrade app to be compatible with Nextcloud 28 (issue #159)

### PR DESCRIPTION
**Pull request by christian-baumann**
Upgrade of the app to the new event API to make it work with Nextcloud 28.

https://help.nextcloud.com/t/symfony-event-dispatcher-update-in-nextcloud-server-28-breaking-changes/167550

### CODE REVIEW

**Comment by Sblop** [comment]
Please approve this modification.

**Comment by devnoname120** [comment]
@susnux Would you mind ✅'ing this PR? Thanks

File: appinfo/info.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
 	<screenshot>https://raw.githubusercontent.com/nextcloud/files_downloadactivity/master/docs/screenshot.png</screenshot>
 
 	<dependencies>
-		<nextcloud min-version="24" max-version="26" />
+		<nextcloud min-version="26" max-version="28" />
```
Comment (nickvergessen) [inline appinfo/info.xml:27]: Should add 29 then as well?

**Review by nickvergessen — APPROVED** [review]

**Comment by devnoname120** [comment]
@nickvergessen Any chance this could be merged? This is spamming my logs.

**Comment by nickvergessen** [comment]
See #135

If anyone is interested in taking over maintainership we can do so.

File: appinfo/info.xml (hunk 1 above)
Comment (skjnldsv) [inline appinfo/info.xml:27]:
```suggestion
		<nextcloud min-version="27" max-version="30" />
```

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**